components/magnolia/config.tsx

```diff
@@ -7,9 +7,13 @@ import Paragraph from '@components/magnolia/components/Paragraph';
 import Expander from '@components/magnolia/components/Expander';
 import List from '@components/magnolia/components/List';
 import Item from '@components/magnolia/components/Item';
+import  TwoColumn from "@components/magnolia/components/TwoColumn";
+import  ThreeColumn from "@components/magnolia/components/ThreeColumn";
 
 const config = {
   componentMappings: {
+    'spa-lm:pages/basic': Basic,
```

Comment:
Well update these as we figure out naming for our base lm. Maybe do it now but def eventually

Reply:
Right! We can chat about it in the morning. I will make it happen before I create the repos as I need to add the light-module as well.

Reply:
updated to use base-lm